.circleci/config.yml

```
@@ -51,78 +55,89 @@ commands:
-            - run: ./tests/scripts/run-different-output-location.sh
-            - persist_to_workspace:
-                root: .
-                paths:
-                    - build.sh
-                    - NuGet-Packed
+                            # special handling for pre-releases (is a constraint by .net Core global tools)
+                            # and also to prevent installation from NuGet-feed (which may have higher version than the
+                            # built tool)
+                            if [[ -z "$ToolVersion" ]]; then
+                                dotnet tool install -g --configfile=ci-nuget.config trx2junit
+                            else
+                                dotnet tool install -g --version="$ToolVersion" --configfile=ci-nuget.config trx2junit
+                            fi
+                    - run:
+                        name: global tool install (trx2junit)
+                        when: on_fail
+                        command: |
+                            echo "Previous job failed, so use trx2junit from NuGet instead of local copy"
+                            echo ""
+                            dotnet tool install -g trx2junit
+                    - run:
+                        name: project test results
+                        when: always
+                        command: |
+                            if [[ -d "tests/TestResults" ]]; then
+                                trx2junit tests/TestResults/*.trx
+                                #rm tests/TestResults/*.trx
+                            fi
+                    - run:
+                        name: code coverage
+                        command: |
+                            if [[ -z "$CODECOV_TOKEN" ]]; then
+                                echo "codecov token not set -- skipping code coverage"
+                            else
+                                ./build.sh coverage
+                            fi
+                    - run: ./tests/scripts/run-samples.sh
+                    - run: ./tests/scripts/run-single-arg.sh
+                    - run: ./tests/scripts/run-multiple-args.sh
+                    - run: ./tests/scripts/run-globbing.sh
+                    - run: ./tests/scripts/run-no-globbing.sh
+                    - run: ./tests/scripts/run-different-output-location.sh
+                    - persist_to_workspace:
+                        root: .
+                        paths:
+                            - build.sh
+                            - NuGet-Packed
             - store_test_results:
                 path: tests/TestResults
             - store_artifacts:
                 path: tests/TestResults
                 destination: TestResults
 
 jobs:
-    build_test_job:
+    release_build_test_job:
         executor: build-executor
         steps:
             - build_test
 
+    debug_build_test_job:
+        executor: build-executor
+        environment:
+            BUILD_CONFIG: debug
```

```suggestion
            BUILD_CONFIG: Debug
```

I think this cause the [build-failures](https://circleci.com/gh/gfoidl/trx2junit/229).